src/transaction/log_meta.hpp

```diff
@@ -35,11 +35,18 @@ namespace cublog
       ~meta () = default;
 
       void load_from_file (std::FILE *stream);       // load meta from meta log file
+      // TODO: function can be called from different threads (main, daemon); must be synched ???
```

Comment:
Is this remark worthwhile?
The code is definitely called from at least 2 different threads, but I am not sure about the timing.
From the "main" thread the code is mainly called in the boot-up/down sequences.
While from the daemon, it is called during regular operation of the server.
There might not be overlap between the two.